src/core/CMakeLists.txt

```diff
@@ -152,15 +152,15 @@ ENDIF (ANDROID)
 
 FIND_PACKAGE(Sqlite3)
 
-INCLUDE_DIRECTORIES(SYSTEM
+TARGET_INCLUDE_DIRECTORIES(qgis_core SYSTEM INTERFACE 
```

Comment:
private?

Reply:
Or let's say: QGIS is an interface dependency
GDAL and PROJ probably not (I think/hope they are only included in cpp files)

Reply:
Ah, thanks!!!!